GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ViewOrganisationSicCodes.cshtml

```diff
@@ -46,83 +46,98 @@
-                    <b>@(organisationSicCode.SicCode.SicCodeId)</b>
-                    @(organisationSicCode.SicCode.Description)
-                    <br />
-                    <span class="govuk-body-s">
-                        part of: @(organisationSicCode.SicCode.SicSection.Description)
-                    </span>
-                </td>
-                <td class="govuk-table__cell">@organisationSicCode.Created.ToString("d MMM yyyy")</td>
-                <td class="govuk-table__cell">@organisationSicCode.Source</td>
+                <th scope="col" class="govuk-table__header">SIC code</th>
+                <th scope="col" class="govuk-table__header">Added on date</th>
+                <th scope="col" class="govuk-table__header">Source</th>
             </tr>
-        }
-    </tbody>
-</table>
+        </thead>
+        <tbody class="govuk-table__body">
+            @foreach (OrganisationSicCode organisationSicCode in currentSicCodes)
+            {
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell">
+                        <b>@(organisationSicCode.SicCode.SicCodeId)</b>
+                        @(organisationSicCode.SicCode.Description)
+                        <br/>
+                        <span class="govuk-body-s">
+                            part of: @(organisationSicCode.SicCode.SicSection.Description)
+                        </span>
+                    </td>
+                    <td class="govuk-table__cell">@organisationSicCode.Created.ToString("d MMM yyyy")</td>
+                    <td class="govuk-table__cell">@organisationSicCode.Source</td>
+                </tr>
+            }
+        </tbody>
+    </table>
+}
 
 <h2 class="govuk-heading-m">Previous SIC codes</h2>
 
-<table class="govuk-table">
-    <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-            <th scope="col" class="govuk-table__header">SIC code</th>
-            <th scope="col" class="govuk-table__header">Added on date</th>
-            <th scope="col" class="govuk-table__header">Removed on date</th>
-            <th scope="col" class="govuk-table__header">Source</th>
-        </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-        @{
-            List<OrganisationSicCode> previousSicCodes = Model.OrganisationSicCodes
-                .Where(osc => osc.Retired.HasValue)
-                .OrderByDescending(osc => osc.Retired.Value)
-                .ThenBy(osc => osc.SicCode.SicCodeId)
-                .ToList();
-        }
-
-        @foreach (OrganisationSicCode organisationSicCode in previousSicCodes)
-        {
+@{
+    List<OrganisationSicCode> previousSicCodes = Model.OrganisationSicCodes
+        .Where(osc => osc.Retired.HasValue)
+        .OrderByDescending(osc => osc.Retired.Value)
+        .ThenBy(osc => osc.SicCode.SicCodeId)
+        .ToList();
+}
+@if (!previousSicCodes.Any())
+{
+    <div class="govuk-body">
+        No SIC codes from Companies House

```

Comment:
I think this should say `No previous SIC codes`.
This list is SIC codes that the organisation used to have, but were removed.
The SIC codes might or might not be from Companies House - so it's a bit confusing to mention CoHo.